multigen/pipes.py

```diff
@@ -188,6 +195,53 @@ def gen(self, inputs):
         return image
 
 
+class MaskedIm2ImPipe(Im2ImPipe):
+    _class = MaskedStableDiffusionImg2ImgPipeline
+
+    def __init__(self, *args, **kwargs):
```

Comment:
`pipe` might be necessary to add here as the explicit argument, while we don't support something like `from_pipe` factory